applications/crossbar/priv/couchdb/schemas/app.json

```diff
@@ -63,6 +63,12 @@
             "required": true,
             "type": "string"
         },
+        "masqueradable": {
```

Comment:
Should be "masquerade-able", non? Or "can_be_masqueraded" or "masquerading_is_allowed"